pkg/util/loop/loop.go

```diff
@@ -31,36 +31,15 @@ type Device struct {
 const (
 	FlagsReadOnly  = 1
 	FlagsAutoClear = 4
-	FlagsPartScan  = 8
```

Comment:
I dont see a reason to remove all of these constants. From my perspective they benefit the package API by having completeness in options available

Reply:
having these constants in our project is a slippery slope at best.    can we find a more authoritative golang package, maintained by kernel developers, that provides these constants?

Reply:
im not aware of any loop package implementation in golang maintained by kernel developers, google or oci container maintainers would be our best hope, but they do not use loop devices in their container lifecycle. We were on our own when developing a lot of this code base because the operations against the OS is not very common in the go community

Reply:
restoring